### PR TITLE
[frameworks][examples] Add Blitz.js

### PR DESCRIPTION
**Pull request by flybayer**
This adds [Blitz.js](https://github.com/blitz-js/blitz) to the zero-config frameworks.

Blitz compiles to a Next.js app, so it uses the same `@now/next` builder.

The change in `packages/now-build-utils/src/detect-builders.ts` was made according to @styfle's suggestion in Slack.

### CODE REVIEW

**Comment by flybayer** [comment]
Looks like most of these failures are from a missing key.

Is there something I need to do?

**Review by styfle — APPROVED** [review]
Thanks!

**Comment by styfle** [comment]
@AndyBitz We don't need an entry in [@now/static-build](https://github.com/zeit/now/blob/master/packages/now-static-build/src/frameworks.ts) because this is using `@now/next`, correct?

File: packages/frameworks/frameworks.json [hunk 1]
```diff
@@ -27,6 +27,34 @@
...
+    },
+    "settings": {
+      "buildCommand": {
+        "placeholder": "`npm run build` or `blitz build`"
+      },
+      "devCommand": {
+        "value": "blitz start"
+      },
+      "outputDirectory": {
+        "value": ".next"
```
Comment (AndyBitz) [inline packages/frameworks/frameworks.json]:
```suggestion
        "placeholder": "Next.js default"
```

So it's equal to Next.js

**Comment by flybayer** [comment]
Hi @AndyBitz, this is ready to go pending your approval :)

**Review by AndyBitz — APPROVED** [review]
Thank you 🥇

**Comment by AndyBitz** [comment]
@styfle @flybayer It looks like the `@vercel/next` package still requires `next` in the dependencies.

When I've deployed the example I've got only a file listing, but when I added `next` to the dependencies it worked. I'll revert for now.

**Pull request merged**